index.ts

```diff
@@ -17,7 +17,7 @@ import {
 } from "@dfinity/nns";
 import { Principal } from "@dfinity/principal";
 import type { Secp256k1PublicKey } from "./src/ledger/secp256k1";
-import { assertLedgerVersion } from "./src/utils";
+import { assertLedgerVersion, isCurrentVersionSmallerThan } from "./src/utils";
```

Comment:
btw. I discovered https://github.com/npm/node-semver if it can be useful for one of your cli one of these days

Reply:
Agree!

I guess that in this library is not so important to avoid using dependencies.

Reply:
Yeah in the cli size matters less to some extension, at least to me